tests/test_core.py

```diff
@@ -4732,6 +4732,13 @@ def test_fs_write(self, js_engines=None):
     out = path_from_root('tests', 'fs', 'test_write.out')
     self.do_run_from_file(src, out, js_engines=js_engines)
 
+  @also_with_noderawfs
```

Comment:
this should best be in `other` as in the other PR. as in there, please see #6473 for a good way to write tests there.

Reply:
Ah, unless we do want to test this with NODERAWFS?